features/seeing_details_of_credit_memo.feature

```diff
@@ -0,0 +1,59 @@
+        And the store has included in price "VAT" tax rate of 20% for "Mugs" within the "US" zone
+        And the store has "Galaxy Post" shipping method with "$10.00" fee
+        And the store allows paying with "Space money"
+        And the store has a product "PHP T-Shirt" priced at "$10.00"
+        And it belongs to "Clothes" tax category
+        And the store has a product "Symfony Mug" priced at "$21.00"
+        And it belongs to "Mugs" tax category
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought a "PHP T-Shirt" and a "Symfony Mug"
+        And the customer "Rick Sanchez" addressed it to "Seaside Fwy", "90802" "Los Angeles" in the "United States" with identical billing address
+        And the customer chose "Galaxy Post" shipping method with "Space money" payment
+        And the order "#00000022" is already paid
+        And I am logged in as an administrator
+
+    @ui @application
+    Scenario: Seeing the details of generated credit memo
+        Given all units from the order "#00000022" are refunded with "Space money" payment
+        When I browse the details of the only credit memo generated for order "#00000022"
+        Then it should have sequential number generated from current date
+        And it should be issued in "United States" channel
+        And it should be issued to "Haas & Milan", "Pacific Coast Hwy", "90003" "Los Angeles" in the "United States" with "1100110011" tax ID
+        And it should be issued from "Rick Sanchez", "Seaside Fwy", "90802" "Los Angeles" in the "United States"
+        And it should contain 1 "PHP T-Shirt" product with "10.00" net value, "1.00" tax amount and "11.00" gross value in "USD" currency
+        And it should contain 1 "Symfony Mug" product with "17.50" net value, "3.50" tax amount and "21.00" gross value in "USD" currency
+        And its subtotal should be "27.50" in "USD" currency
+        And it should have a tax item "US VAT (10%)" with amount "1.00" in "USD" currency
+        And it should have a tax item "VAT (20%)" with amount "3.50" in "USD" currency
+        And its total should be "32.00" in "USD" currency
+
+    @ui @application
+    Scenario: Seeing the details of generated credit memo with partial price
+        Given 1st "PHP T-Shirt" product from order "#00000022" has already "$5.00" refunded with "Space money" payment
+        When I browse the details of the only credit memo generated for order "#00000022"
+        Then it should have sequential number generated from current date
+        And it should be issued in "United States" channel
+        And its subtotal should be "5.00"
+        And it should contain 1 "PHP T-Shirt" product with "5.00" net value, "0.50" tax amount and "5.50" gross value in "USD" currency
+        And it should have a tax item "US VAT (10%)" with amount "0.50" in "USD" currency
+        And its total should be "5.50" in "USD" currency
```

Comment:
Why does this step differ from [this one](https://github.com/Sylius/RefundPlugin/pull/179/files#diff-bfa80ae7010f420ce914820afb19dbf8R47) regarding currency?

Reply:
I don't understand the question, subtotal is a net value, but total gross value, probably I should change the naming

Reply:
And its subtotal should be "5.00" 
And its total should be "5.50" **in "USD" currency** 

imo it should be either:
```
And its subtotal should be "5.00" 
And its total should be "5.50"
```

or 

```
And its subtotal should be "5.00" in "USD" currency
And its total should be "5.50" in "USD" currency
```

it looks inconsistent now.